manila/share/drivers/netapp/dataontap/client/api.py

```diff
@@ -70,6 +70,7 @@
 ESCOPENOTFOUND = '18259'
 ESVMDR_CANNOT_PERFORM_OP_FOR_STATUS = '18815'
 OPERATION_ALREADY_ENABLED = '40043'
+OPERATION_ALREADY_ACTIVE = '40043'
```

Comment:
Do we really need two different names for the same error code number? (see the line above)

Reply:
thought of having different name for more readability of ACTIVE vs ENABLED. but seems both are almost same in meaning, so removed.